packages/remix-edge-adapter/src/server.ts

```diff
@@ -38,6 +38,10 @@ export function createRequestHandler({
       const loadContext = (await getLoadContext?.(request, context)) || context
 
       const response = await remixHandler(request, loadContext)
+
+      // A useful header for debugging
+      response.headers.set('x-nf-runtime', 'Edge')
```

Comment:
As discussed in https://github.com/netlify/remix-compute/pull/16#discussion_r1099884242 @ascorbic, we now have the `x-nf-runtime` header.

![CleanShot 2023-02-08 at 08 59 25](https://user-images.githubusercontent.com/833231/217551415-cec2f5ff-7295-410d-b0da-96cb5d64bf0f.png)